rust/agama-lib/src/product/client.rs

```diff
@@ -158,7 +162,7 @@ impl<'a> ProductClient<'a> {
             .registration_proxy
             .register_addon(
                 &addon.id,
-                &addon.version,
+                &addon.version.clone().unwrap_or_default(),
```

Comment:
Would an empty version work?

Reply:
Yes, in the end the backend receives empty version because `nil` cannot be sent over DBus. So omitting the version is the same as using empty string.

And if SCC really uses an empty version then it will work too, just using a slightly more complicated path in the code.